_data-prepper/getting-started.md

```diff
@@ -28,10 +28,39 @@ If you have special requirements that require you to build from source, or if yo
 
 ## 2. Configuring Data Prepper
 
-You must configure Data Prepper with a pipeline before running it. You'll modify the following files:
+Two configuration files are required to run a Data Prepper instance. Optionally, you can configure a Log4j 2 configuration file. See [Configuring Log4j]({{site.url}}{{site.baseurl}}/data-prepper/managing-data-prepper/configuring-log4j/). The following list describes the purpose of each configuration file:
 
-* `data-prepper-config.yaml`
-* `pipelines.yaml`
+Option | Required | Description
+:--- | :--- | :--- |
+`pipelines.yaml` | Yes | This file describes which data pipelines to run, including sources, processors, and sinks. |
+`data-prepper-config.yaml` | Yes | This file contains Data Prepper server settings that allow you to interact with exposed Data Prepper server APIs. |
+`log4j2-rolling.properties` | No | This file contains Log4j 2 configuration options and can be a JSON, YAML, XML, or a .properties file type. |
+
+For Data Prepper previous to version 2.0, the `.jar` file expects the pipeline configuration file path to be followed by the server configuration file path. See the following configuration path example.
+
+```
+java -jar data-prepper-core-$VERSION.jar pipelines.yaml data-prepper-config.yaml
+```
+
+Optionally, you can add `"-Dlog4j.configurationFile=config/log4j2.properties"` to the command to pass a custom Log4j 2 configuration file. If you don't provide a properties file, Data Prepper defaults to the `log4j2.properties` file in the `shared-config` directory.
+
+
+Starting with Data Prepper 2.0, you can launch Data Prepper by using the following `data-prepper` script that does not require any additional command line arguments:
+
+```
+bin/data-prepper
+```
+
+Configuration files are read from specific subdirectories in the application's home directory:
+1. `pipelines/`: Used for pipelines configurations. Pipelines configurations can be written in one or more yaml files.
```

Comment:
I don't think "pipelines" plural is a proper noun here. The way I read this is that `pipelines/` indicates you can list more than one configuration files, but you can also have just one.
We refer to them singularly throughout our OpenSearch documentation as a single pipeline or multiple pipelines. So you could have one pipeline configurations or many pipeline configurations. In other words, I think "Used for pipeline configurations" is appropriate.

YAML